ivy/functional/ivy/creation.py

```diff
@@ -2002,7 +2002,7 @@ def one_hot(
     >>> ivy.one_hot(x, 5, out=z)
     ivy.array([[1., 0., 0., 0., 0.]])
     >>> print(z)
-    ivy.array([[1., 0., 0., 0., 0.]])
+    ivy.array([[1., 0., 0., 0., 0.], [1., 0., 0., 0., 0.]])
```

Comment:
this seems a bit unusual, the `out` argument should ideally have the same values as the output of the function. Could you please check this one?

Reply:
Yup 👍🏻. Corrected it.

Reply:
awesome, thanks!